config/webpack.target.browser.js

```diff
@@ -15,7 +15,8 @@ module.exports = {
   },
   plugins: [
     new webpack.DefinePlugin({
-      __TARGET__: JSON.stringify('browser')
+      __TARGET__: JSON.stringify('browser'),
+      __POUCH__: JSON.stringify(process.env.FORCE_POUCH ? true : false)
```

Comment:
Are you sure that you want to get a string here ? It means that you can't just write `if (__POUCH__)`, since it will always be a non empty string, so truthy, no?

Reply:
Oh, OK thanks !